script.js

```diff
@@ -1,12 +1,138 @@
-//You can edit ALL of the code here
```

Comment:
In general, we want to avoid putting variables in the global scope if possible. Otherwise, there's a chance you may accidentally change the variable without realising (like what happened when I debugged with you last week).

The `container` variable is only used on line 38, so we should move it into the scope of that function (`makePageForEpisodes`).

For the `searchContainer` variable, you have an easy way to retrieve it (by calling `document.getElementById("search-container");`). I would suggest in each of the functions you use it (`searchForEpisode` and `makeDropdownMenu`) that you retrieve it and assign it to a new variable at the beginning of the function. It will exist only in the function scope, and then it will be removed from memory.

E.g.
```
function searchForEpisode() {
let searchContainer = document.getElementById("search-container");
// There rest of your code below

}

```